### DISCUSSION TITLE
One MX and multiple domains configuration

### DISCUSSION THREAD

**Discussion by lakano** [emersion/maddy#597] [Q&A]
Hi!
First, thank to all contributors for this amazing project!

I have a maddy server where multiple domains will be handle per the same FQDN mx.domainA.email 
The primary_domain is domainA.email and I have few others extra domains like domainB.com domainC.com domainC.fr domainC.net

1) each extra domains need to also host a .well-known/mta-sts.txt file or is it only for the primary_domain ?

2) I've found possibilty to share the same cred and/or imap-acct for all the domains, but in my case I would like to do it only for a selected list of domain ( domainC.com domainC.fr domainC.net is the same hostname with different TLDs)

Is it possible ?

Thanks!

**Comment by Damaj301damaj-lol** [comment]
1. All domains must have an mta-sts subdomain from my experience, also since the value doesn't change, you can have a single mta-sts folder and reroute all mta-sts.*.* to it
2. I also have mx.domainA.tech and domainB.com, so you can actually go to the initial setup page on maddy.email, and ignore the installation, just follow the DNS part (especially the DKIM, SPF, and "mx.domainA.tech handles mail for domainB.com")
Then create as you would, same hostname and different TLDs usually count as seperate entities. So you can ```maddy creds create whatever@domainA/B/C/D.whatever```
And that's how i have maddy setup for 2 domains

**Comment by lakano** [comment]
Thank you @Damaj301damaj-lol for your feedback
1- Understood
2- May be I bad explained my problem of not understood your solution.
I have 5 domains:
domainA.email => owning the mx for every domains ( mx.domainA.email ) and have an independent contact@
domainB.com => an independent contact@
domainC.com + domainC.fr + domainC.net => all the same contact@ of the same creds/imap

Is it possible?
Thanks
